packages/animation/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@web-stories-wp/animation",
+  "private": true,
+  "type": "module",
+  "engines": {
+    "node": ">= 15",
+    "npm": "~7"
+  },
+  "exports": {
+    "default": "./src/index.js"
+  },
+  "main": "./src/index.js",
+  "dependencies": {
+    "@web-stories-wp/design-system": "*",
```

Comment:
This is the dependency I'd want to avoid! The animation package should not depend on the design system just for using `createContext`. I doubt this will be treeshakeable.

We might need the dedicated `@web-stories-wp/react` package sooner or later...

Reply:
Yeah, this is a problem elsewhere in the code as well. I think this should be happened elsewhere.

Reply:
I mentioned this issue in https://docs.google.com/document/d/1ImRe9wYKWGCAZiq3jro5KLpSJK_NMKXHC6FWqs3nwl0/edit

Seems reasonable to create a dedicated package for this. I'll open a ticket.

Reply:
Ticket created: #8159